securemeet.py

```
@@ -0,0 +1,907 @@
+                to_purge = c.execute('SELECT id, created_at, audio_path, transcript_path FROM meetings').fetchall()
+                for mid, created, aud, tr in to_purge:
+                    cdt = datetime.fromisoformat(created)
+                    if aud and cdt < aud_cut:
+                        try:
+                            os.remove(aud)
+                            c.execute('UPDATE meetings SET audio_path=NULL WHERE id=?', (mid,))
+                        except Exception:
+                            pass
+                    if cdt < tr_cut:
+                        try:
+                            if tr: os.remove(tr)
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)